src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -6,6 +6,7 @@
  */
 package com.connexta.store.controllers;
 
+import static org.springframework.http.HttpStatus.*;
```

Comment:
There is something about IntelliJ converting imports to asterisks. Why doesn't Spotless doesn't fix these * imports?  If is bugs people I can go back and use explicit imports.

Reply:
I think we should use explicit imports.

Reply:
Here are my settings in Intellij
![image](https://user-images.githubusercontent.com/8041246/67905048-7a920200-fb2d-11e9-88da-55c44aa110dc.png)

Reply:
I believe the company coding standards state using explicit imports.